lib/modules/apostrophe-salesforce-connect/index.js

```diff
@@ -115,18 +111,17 @@ module.exports = {
                 return next(err);
               }
 
-              if (!user) return next('No user authenticated.');
-
-              req.session.sfUser = await connectSalesforce(user, sfExpOptions) || null;
-
+              if (!user) {
+                return next('No user authenticated.');
+              }
+              const sfe = self.apos.modules['apostrophe-salesforce-experience'];
+              await sfe.completeProfile(user);
```

Comment:
I assume this is new-to-me information. This doesn't need to be `user = await sfe.completeProfile(user)`?